Bottom: c4ccc5e8023c3fdd52548e3e38657ac71cf3c693
Top:    48e008a226119948df95bda2046812bb8b23d9d5
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-02-24 02:28:46 -0300

adjust copyright notice

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index e0c2a6a..32f9702 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -1,7 +1,8 @@
-/* Library interface to C front end
-   Copyright (C) 2014 Free Software Foundation, Inc.
+/* Library interface to C++ front end.
+   Copyright (C) 2014-2016 Free Software Foundation, Inc.
 
-   This file is part of GCC.
+   This file is part of GCC.  As it interacts with GDB through libcc1,
+   they all become a single program as regards the GNU GPL's requirements.
 
    GCC is free software; you can redistribute it and/or modify it under
    the terms of the GNU General Public License as published by the Free
